cli/tests/unit/dispatch_json_test.ts

```diff
@@ -19,9 +19,11 @@ unitTest(
   }
 );
 
-/* eslint-disable @typescript-eslint/no-namespace, @typescript-eslint/no-explicit-any,no-var */
+/* eslint-disable @typescript-eslint/no-explicit-any,no-var */
```

Comment:
@bartlomieju the issue I get is that if I remove this ignore, `/tools/lint.py` yells about the `any` and `var`, but I can't add it above `var core: any;` otherwise the `deno-lint-ignore` won't be effective and then `deno lint` fails. So this seems the only way to get both linters to pass.

Reply:
It should be possible to do: `// eslint-disable-next-line @typescript-eslint/no-explicit-any no-var`

If that doesn't work then we should update linter to recongnize ignore directive separated by commas

Reply:
Seems `eslint` requires multiple directives to be comma-separated — added an issue in `deno_lint`: https://github.com/denoland/deno_lint/issues/179

I will look into a PR for `deno_lint`. Otherwise, the hacky solution of combining `eslint-disable-next-line` and `eslint-disable-line` to disable both rules also works.